app/controllers/ems_block_storage_controller.rb

```diff
@@ -15,7 +16,7 @@ def self.model
   end
 
   def self.table_name
-    @table_name ||= "ems_block_storage"
+    @table_name ||= "ems_storage"
```

Comment:
Wouldn't this break any other block storage providers?

Reply:
We dont have block_storage_manager or object_storage_manager, only storage_manager with mixin for block_storage or object storage. This is why self.model is set to ManageIQ::Providers::StorageManager, and I guess when I  rewrite the code I change it because of the same logic. As far as I can see table_name is mostly affecting button actions and it should work with other block storage providers (I also tested it). But after thinking on it again I agree with you that it should be ems_block_storage. I changed it back and also fixed the relevant code under the buttons.